src/components/Admin/Admin.scss

```diff
@@ -10,7 +10,3 @@
         flex: 1;
     }
 }
-
-.download-btn div {
```

Comment:
With the most recent Paragon release, the `div` in the `Icon` component was changed to a `span` so this CSS is no longer needed.